Neos.Flow/Classes/Cache/CacheManager.php

```diff
@@ -62,6 +63,12 @@ class CacheManager
      */
     protected $persistentCaches = [];
 
+    /**
+     * The ReflectionService is only needed during compile time for handling
```

Comment:
```suggestion
     * The ReflectionService is only needed during compile time for handling flushConfigurationCachesByCompiledClass
```
Writibus interruptus...